.drone.yml

```diff
@@ -11,12 +11,12 @@ concurrency:
 
 steps:
 - name: sync
-  image: xoxys/git-batch:latest
+  image: thegeeklab/git-batch:latest
   commands:
-  - git-batch
+  - git-batch -vvv
```

Comment:
`-vvv` will switch to debug mode, `-vv` would be enough for info and warning (just as information) up to you what level to choose

Reply:
ok

Reply:
done